efel/pyfeatures/pyfeatures.py

```diff
@@ -28,39 +28,62 @@
 
 import numpy
 import efel.cppcore
+from scipy.ndimage.filters import gaussian_filter1d
```

Comment:
maybe we should only import this in the feature itself, so that scipy doesn't become a hard dependency